Meta/Web Foundations team responsibilities.md

```diff
@@ -0,0 +1,67 @@
+  2. Any team feeling that our recommended best practices or libraries prevent them from being capable of delivering a user-facing feature should ignore the recommendation and bring this up with us.
+* It **must** be a topic that affects a majority of applications. The Web Foundation team will not build libraries for that benefit only a small handful of projects.
+* It **must not** be a topic that has universal consensus in the industry.
+
+**At least one** of the following criteria must also be met for the Web Foundation team to get involved:
+
+* It affects a key aspect of the user experience (performance, accessibility, etc) and is difficult for a typical developer to adequately address.
+* It has a material impact on the scalability of an application.
+* It spans multiple layers of the stack, most notably when it is a problem that has implications for build and runtime.
+* It involves a connection to a shared foundational technology between mobile and web, most notably to Polaris and GraphQL.
+
+### Application of involvement
+
+With these criteria in mind, the Web Foundations team is responsible for: 
+
+* **Build.** The build process for applications has an important impact on performance, as ineffective splitting and loading of assets can delay interactivity. Asynchronous loading of code also requires coordination of tooling, server, and client code. We consolidate our build tooling in [sewing-kit](https://github.com/Shopify/sewing-kit).
+* **Testing.** Testing is a key part of creating a scalable application that can be effectively shared/ transferred between developers. We provide recommendations on testing tools and best practices in this repo, and include a solid default testing experience in [sewing-kit](https://github.com/Shopify/sewing-kit).
+* **GraphQL.** GraphQL is vitally important to all applications at Shopify. In order to maintain a consistent approach between clients across platforms, the Web Foundation team coordinates its best practices and tooling with those of the mobile foundation team. Handling GraphQL effectively also involves tight integration with build, test, and type checking/ linting. We collect our GraphQL tooling in [graphql-tools-web](https://github.com/Shopify/graphql-tools-web), and provides an excellent default experience for GraphQL in [sewing-kit](https://github.com/Shopify/sewing-kit).
+
+These responsibilities are summarized in the diagram below.
+
+<figure>
+  <img src="./images/Responsibilities%20-%20Shopify.png" alt="Web Foundation responsibilities for Shopify" />
+  <figcaption>A summary of components that the Web Foundation team is responsible for (the bottom four layers, highlighted in blue)</figcaption>
+</figure>
+
+## Maintenance of foundational piece of Shopify Web
+
+![Web Foundation responsibilities for Shopify Web](./images/Responsibilities%20-%20Web.png)
```

Comment:
does this need a `figcaption` as well for the asterisk next to `Polaris`?

Reply:
Was wondering about the asterisk as well.